src/aap_eda/core/models/source.py

```diff
@@ -16,36 +16,16 @@
 
 from django.db import models
 
-from aap_eda.core.enums import RestartPolicy
+from aap_eda.core.models.activation import Activation
 
-__all__ = "Source"
+__all__ = ["Source"]
 
 
-class Source(models.Model):
+class Source(Activation):
```

Comment:
Sources are not activations.   They are something else.   Maybe both sources and activations are processes.